src/patternfly/components/Button/button.scss

```diff
@@ -362,14 +362,6 @@
         border-bottom-width: var(--pf-c-button--m-control--m-expanded--after--BorderBottomWidth);
       }
     }
-
-    &:disabled,
-    &.pf-m-disabled {
-      --pf-c-button--m-control--after--BorderTopColor: var(--pf-c-button--m-control--disabled--after--BorderTopColor);
-      --pf-c-button--m-control--after--BorderRightColor: var(--pf-c-button--m-control--disabled--after--BorderRightColor);
-      --pf-c-button--m-control--after--BorderBottomColor: var(--pf-c-button--m-control--disabled--after--BorderBottomColor);
```

Comment:
I think you still need border-bottom-color
without `--pf-c-button--m-control--after--BorderBottomColor`
![Screen Shot 2020-05-14 at 1 00 40 PM](https://user-images.githubusercontent.com/5385435/81963456-1a534100-95e3-11ea-8c7c-547e92fcba56.png)

versus

![Screen Shot 2020-05-14 at 1 00 50 PM](https://user-images.githubusercontent.com/5385435/81963478-23441280-95e3-11ea-810c-2a628515bf23.png)

Reply:
The intended design is no bottom border: https://github.com/patternfly/patternfly/pull/3049

You can see an example here verified with design and a user how it should look in product (with no bottom border) - https://github.com/patternfly/patternfly/issues/2997